src/custom/components/AccountDetails/Transaction.tsx

```diff
@@ -337,10 +343,20 @@ function ActivitySummary(params: {
               <b>Limit price</b>
               <i>{orderSummary.limitPrice}</i>
             </SummaryInnerRow>
-            <SummaryInnerRow>
-              <b>Valid to</b>
-              <i className={isCancelled ? 'cancelled' : ''}>{orderSummary.validTo}</i>
-            </SummaryInnerRow>
+
+            {!orderSummary.fulfillmentTime && (
```

Comment:
If the `fulfillmentTime` does not exist (`undefined` by default) it skips showing the `validTo`.